conductr-network-ec2.yaml

```diff
@@ -0,0 +1,213 @@
+---
+AWSTemplateFormatVersion: "2010-09-09"
+
+Description: >
+  Create Lightbend Production Suite demo EC2 network and load balancer
+
+Metadata:
+  AWS::CloudFormation::Interface:
+    ParameterGroups:
+      - Label:
+          default: Network Configuration
+        Parameters:
+          - VpcCidrBlock
+          - SubnetCidrBlocks
+
+Parameters:
+  VpcCidrBlock:
+    Type: String
+    Default: "10.23.0.0/16"
+    Description: The CIDR block for the new VPC. Choose a block that does not overlap with any other VPC in your account.
+    AllowedPattern: "(\\d{1,3})\\.(\\d{1,3})\\.(\\d{1,3})\\.(\\d{1,3})/(\\d{1,2})"
+    ConstraintDescription: must be a valid VPC CIDR block (see http://docs.aws.amazon.com/AmazonVPC/latest/UserGuide/VPC_Subnets.html)
+  SubnetCidrBlocks:
+    Type: CommaDelimitedList
+    Default: "10.23.0.0/24, 10.23.1.0/24, 10.23.2.0/24"
+    Description: CIDR blocks for the three subnets, separated by commas. These must fall within the VPC CIDR block.
+
+Resources:
+  # The network: a VPC, three public subnets, and a route to the internet
+  VPC:
+    Type: AWS::EC2::VPC
+    Properties:
+      CidrBlock: !Ref VpcCidrBlock
+      EnableDnsSupport: true
+      EnableDnsHostnames: true
+      InstanceTenancy: default
+      Tags:
+        - Key: Name
+          Value: Lightbend Cluster VPC
+  InternetGateway:
+    Type: AWS::EC2::InternetGateway
+    Properties:
+      Tags:
+        - Key: Name
+          Value: Lightbend Cluster Internet Gateway
+  VPCGatewayAttachment:
+    Type: AWS::EC2::VPCGatewayAttachment
+    Properties:
+      VpcId: !Ref VPC
+      InternetGatewayId: !Ref InternetGateway
+  Subnet0:
+    Type: AWS::EC2::Subnet
+    Properties:
+      VpcId: !Ref VPC
+      CidrBlock: !Select [ 0, !Ref SubnetCidrBlocks ]
+      AvailabilityZone: !Select [ 0, !GetAZs "" ]
+      Tags:
+        - Key: Name
+          Value: !Sub
+            - "Lightbend Cluster Subnet ${AZ}"
+            - AZ: !Select [ 0, !GetAZs "" ]
+  Subnet1:
+    Type: AWS::EC2::Subnet
+    Properties:
+      VpcId: !Ref VPC
+      CidrBlock: !Select [ 1, !Ref SubnetCidrBlocks ]
+      AvailabilityZone: !Select [ 1, !GetAZs "" ]
+      Tags:
+        - Key: Name
+          Value: !Sub
+            - "Lightbend Cluster Subnet ${AZ}"
+            - AZ: !Select [ 1, !GetAZs "" ]
+  Subnet2:
+    Type: AWS::EC2::Subnet
+    Properties:
+      VpcId: !Ref VPC
+      CidrBlock: !Select [ 2, !Ref SubnetCidrBlocks ]
+      AvailabilityZone: !Select [ 2, !GetAZs "" ]
+      Tags:
+        - Key: Name
+          Value: !Sub
+            - "Lightbend Cluster Subnet ${AZ}"
+            - AZ: !Select [ 2, !GetAZs "" ]
```

Comment:
Should we enable MapPublicIpOnLaunch. Newbies using these SN's afterwards are likely to miss the 'enable public ip' box at launch.